OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -203,7 +203,7 @@ public bool AircraftCanEnter(Actor a)
 
 		public bool IsLeavingCell(CPos location, SubCell subCell = SubCell.Any) { return false; } // TODO: Handle landing
 		public SubCell GetValidSubCell(SubCell preferred) { return SubCell.Invalid; }
-		public SubCell GetAvailableSubCell(CPos a, SubCell preferredSubCell = SubCell.Any, Actor ignoreActor = null, bool checkTransientActors = true) { return SubCell.Invalid; } // Does not use any subcell
+		public SubCell GetAvailableSubCell(CPos a, SubCell preferredSubCell = SubCell.Any, Actor ignoreActor = null, bool checkTransientActors = true) { return SubCell.Any; } // Does not use any subcell
```

Comment:
I assume that's caused by [this check](https://github.com/OpenRA/OpenRA/pull/7468/files#diff-86d6fc75cf06769f960cd874106931f7R125) then.